bundle/manifests/storageosoperator.clusterserviceversion.yaml

```diff
@@ -564,7 +567,8 @@ spec:
   minKubeVersion: 1.13.0
   provider:
     name: StorageOS, Inc
+  replaces: storageosoperator.v2.3.3
```

Comment:
Replaces is not needed in the new bundle format.

Reply:
Removed - thanks